<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemEditorDialog</class>
 <widget class="QDialog" name="ItemEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="3" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabMain">
      <attribute name="title">
       <string>&amp;Основная информация</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="6" column="1">
        <widget class="QLineEdit" name="edtNewServicePrefix">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QRadioButton" name="rbModifyByRegExp">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Регулярное выражение</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QRadioButton" name="rbRemoveService">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Удаляет услугу</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QRadioButton" name="rbModifyService">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Изменяет префикс кода услуги</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0">
        <widget class="QRadioButton" name="rbReplaceService">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Заменяет код услуги на</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="rbNoModifyService">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Не меняет услугу</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="edtNewServiceName">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QPushButton" name="btnRegExpTest">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Испытание</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" rowspan="2">
        <widget class="QPlainTextEdit" name="edtRegExp">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2';&quot;&gt;Регулярное выражение задаётся в виде&lt;br/&gt;/&lt;/span&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-weight:600;&quot;&gt;искомое&lt;/span&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2';&quot;&gt;/&lt;/span&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-weight:600;&quot;&gt;заменяющее&lt;/span&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2';&quot;&gt;/&lt;br/&gt;при этом используются регулярные выражения Qt.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2';&quot;&gt;Можно задать несколько выражений, по выражению на строку. В этом случае делаются последовательные попытки замены, после первой успешной замены попытки прекращаются.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblRegionalCode">
         <property name="text">
          <string>&amp;Региональный код</string>
         </property>
         <property name="buddy">
          <cstring>edtRegionalCode</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Наименование</string>
         </property>
         <property name="buddy">
          <cstring>edtName</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblCode">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Код</string>
         </property>
         <property name="buddy">
          <cstring>edtCode</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="edtCode"/>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="edtName"/>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="edtRegionalCode"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabIdentification">
      <attribute name="title">
       <string>&amp;Идентификация</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="CInDocTableView" name="tblIdentification"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>edtCode</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>edtRegionalCode</tabstop>
  <tabstop>rbNoModifyService</tabstop>
  <tabstop>rbRemoveService</tabstop>
  <tabstop>rbReplaceService</tabstop>
  <tabstop>edtNewServiceName</tabstop>
  <tabstop>rbModifyService</tabstop>
  <tabstop>edtNewServicePrefix</tabstop>
  <tabstop>rbModifyByRegExp</tabstop>
  <tabstop>edtRegExp</tabstop>
  <tabstop>btnRegExpTest</tabstop>
  <tabstop>tblIdentification</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>rbReplaceService</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtNewServiceName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbModifyService</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtNewServicePrefix</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbModifyByRegExp</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnRegExpTest</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbModifyByRegExp</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtRegExp</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
